### PR TITLE
Add complication to chameleon card#192

### PR DESCRIPTION
**Pull request by jbforth4**

### CODE REVIEW

File: packages/card/src/chameleon-card-style.ts [hunk 1]
```diff
@@ -16,4 +16,22 @@ export default css`
...
+
+  :host([complication]) {
+    padding-top: 35px;
+    position: relative;
+  }
+
+  .complication {
+    background-color: var(--color-secondary, #69c9b9);
+    border-radius: var(--border-radius, 0.5rem) 0 var(--border-radius, 0.5rem) 0;
+    color: var(--color-primary-light, #679dea);
```
Comment (ryuhhnn) [inline packages/card/src/chameleon-card-style.ts]: Remove this line

File: packages/card/src/chameleon-card-style.ts [hunk 2]
```diff
@@ -16,4 +16,22 @@ export default css`
...
+    font-size: var(--font-size-label, 0.875rem);
+    padding: 4px 16px;
+    color: var(--color-surface, #ffffff);
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 50px;
```
Comment (ryuhhnn) [inline packages/card/src/chameleon-card-style.ts]: Can you remove the `width` property? It's unneeded

File: .storybook/stories/card.stories.ts [hunk 3]
```diff
@@ -153,3 +153,28 @@ stories.add(
...
+    const titleText = text("Title", "St. Louis, MO");
+    const complicationText = text("Complication Text", "Sup??");
+
+    return html`
+      <style>
+        chameleon-card {
+          max-width: 400px;
+        }
+      </style>
+      <chameleon-card .complication=${complicationText}>
```
Comment (ryuhhnn) [inline .storybook/stories/card.stories.ts]: Can you add the `rounded` attribute to this?

File: packages/card/src/chameleon-card-style.ts (hunk 1 above)
Comment (jbforth4) [reply packages/card/src/chameleon-card-style.ts]: which line specifically? The color?

Comment (ryuhhnn) [reply packages/card/src/chameleon-card-style.ts]: Yes, that's correct! It's a duplicate

File: .storybook/stories/card.stories.ts (hunk 3 above)
Comment (jbforth4) [reply .storybook/stories/card.stories.ts]: is this what you were looking for: 
      `<style>
        chameleon-card {
          max-width: 400px;
          border-radius: var(--border-radius, 0.5rem);
        }
      </style>`

Comment (ryuhhnn) [reply .storybook/stories/card.stories.ts]: Nope, just change the markup to read: `<chameleon-card .complication="${complication text}" rounded>`

**Pull request merged**